src/components/Cell.vue

```diff
@@ -12,7 +12,18 @@ td.cell.noselect(
   @mouseup='$emit("mouseup", $event)'
 )
   span.editable-field(v-if='cellEditing[0] === rowIndex && cellEditing[1] === columnIndex')
+    select(v-if="(inputType === 'select')"
+      ref="input"
+      @keyup.enter="setEditableValue",
+      @keydown.tab="setEditableValue",
+      @change="setEditableValue",
+      @keyup.esc="editCancelled",
+      @focus="editPending = true"
+      @blur="leaved"
+      )
+        option(v-for="op in column.selectOptions" :value="op.value") {{op.text}}
```

Comment:
```suggestion
    )
      option(v-for='option in column.selectOptions' :key='option.value' :value='option.value') {{ op.text }}
```

Reply:
option(v-for="(option, index) in column.selectOptions" :key='option.value+index' :value="option.value") {{option.text}}

what do you think of adding index to handle duplicate values with different text?

Reply:
I prefer only index, because `option.value+index` could launch unexpected results if option.value is numeric. What do you think?